.github/workflows/translate.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run:  bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/en.lproj && tx push -s --skip"
 
       - name: l10n-pull
-        run: bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/ && tx pull --force --skip --all"
+        run: bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "tx pull -a --skip --minimum-perc=75 -f"
```

Comment:
please use the --long args rest looks fine thx

Reply:
fixed